events/nickname_events/nickname_events.txt

```diff
@@ -962,10 +962,17 @@ lifestyle_nicknames.0009 = { # Learning nickname for someone who's arrogant
 		}
 	}
 	
-	option = { # Truthseeker
-		name = lifestyle_nicknames.0009.b
-		
-		set_nickname_effect = { NICKNAME = nick_the_truthseeker }
+	option = { # Truthseeker or Seeker
+		name = lifestyle_nicknames.0009.b.wc
+
+		random_list = {
+			50 = {
+				set_nickname_effect = { NICKNAME = nick_the_truthseeker }
+			}
+			50 = {
+				set_nickname_effect = { NICKNAME = nick_the_seeker }
+			}
+		}
```

Comment:
To keep Old Gods' vibe.
```suggestion
		if = {
			limit = { faith = { is_in_family = rf_shadow } }
			set_nickname_effect = { NICKNAME = nick_the_seeker }
		}
		else = {
			set_nickname_effect = { NICKNAME = nick_the_truthseeker }
		}
```

Reply:
It's not about the Old Gods in CK2. Seems to me like this is a title that can be achieved with success in wow.

Reply:
> It's not about the Old Gods in CK2. Seems to me like this is a title that can be achieved with success in wow.

Mmmkay.